corehq/apps/users/management/commands/reset_demo_users_restores.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.ota.utils import reset_demo_user_restore
+from corehq.apps.users.dbaccessors import get_practice_mode_mobile_workers
+from corehq.apps.users.models import CommCareUser
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Reset restores for demo users"
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            default=False,
+            help='Do not actually reset, just verbosely log what will happen',
+        )
+
+    def handle(self, domain, dry_run, **options):
+        if not dry_run:
+            confirmation = input("Please confirm you want to reset for real?(y)")
```

Comment:
nit:
```suggestion
            confirmation = input("Please confirm you want to reset for real? [y/N]")
```

Reply:
I was thinking of y is yes and everything else is no.

Reply:
Looking at our other management commands it seems like we use `(y/n)` or `[y/n]` with or without the capital `n`/`N`, even where 'n' means the same as "anything that isn't 'y'". I think it'd be nice to follow that convention.

Reply:
+1 for displaying this convention. This is conventional both in HQ and on the command line in general.

```bash
commcare-hq $ git grep '\[[yY]/[nN]]' | wc -l
25
```

"(y)" could be ambiguous. People might think that it means that "y" is the only option.